django_statsd/clients/null.py

```diff
@@ -4,5 +4,8 @@
 class StatsClient(StatsClient):
     """A null client that does nothing."""
 
+    def __init__(self, host, port, prefix=None, **kwargs):
```

Comment:
Not sure why this is required. Any idea ?

Reply:
`StatsClient.__init__` sets up a socket connection to the StatsD host. The null client doesn't need that. On the other hand, StatsClient expects the `_prefix` attribute.